float32[] robot_state
int32 current_action 
---
float32[] next_action_prob
